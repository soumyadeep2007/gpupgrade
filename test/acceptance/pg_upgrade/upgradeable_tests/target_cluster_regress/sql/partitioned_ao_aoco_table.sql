-- Copyright (c) 2017-2021 VMware, Inc. or its affiliates
-- SPDX-License-Identifier: Apache-2.0

--------------------------------------------------------------------------------
-- Validate that the upgradeable objects are functional post-upgrade
--------------------------------------------------------------------------------
SELECT * FROM p_ao_table_with_multiple_segfiles;
SELECT * FROM p_ao_table_with_multiple_segfiles;
