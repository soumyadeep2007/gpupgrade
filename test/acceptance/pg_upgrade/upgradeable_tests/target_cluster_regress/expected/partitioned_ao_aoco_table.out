-- Copyright (c) 2017-2021 VMware, Inc. or its affiliates
-- SPDX-License-Identifier: Apache-2.0

--------------------------------------------------------------------------------
-- Validate that the upgradeable objects are functional post-upgrade
--------------------------------------------------------------------------------
SELECT * FROM p_ao_table_with_multiple_segfiles;
 id | name    
----+---------
 4  | Carolyn 
 2  | Jane    
 2  | Jane    
 1  | Jane    
 1  | Jane    
(5 rows)
SELECT * FROM p_ao_table_with_multiple_segfiles;
 id | name    
----+---------
 2  | Jane    
 2  | Jane    
 1  | Jane    
 1  | Jane    
 4  | Carolyn 
(5 rows)
